Source/Onboarding/OnboardingStep.swift

```diff
@@ -71,6 +85,43 @@ class OnboardingStep: NSObject {
                 case .start: return .start
             }
         }
+
+        var analyticsStep: Analytics.OnboardingStep {
```

Comment:
Maybe we could make `Analytics.OnboardingStep` have a String rawValue, and then this function could be something like `Analytics.OnboardingStep(rawValue: name.rawValue) ?? .invalid`.

Reply:
👍